docs/index.md

```diff
@@ -0,0 +1,80 @@
+---

```

Comment:
Is it possible to include markdown files from outside of the folder? It would be nice if this file would also be displayed as README.md in the project root.

Reply:
I also think so, but dind't found a solution. I think about a solution.

Reply:
Maybe it's possible with symbolic links. Will check this the next days